packages/components/spinner/src/Spinner.doc.mdx

```diff
@@ -0,0 +1,58 @@
+import { ArgsTable, Meta, Story, Canvas } from '@storybook/addon-docs'
+import { StoryHeading } from '@docs/helpers/StoryHeading'
+
+import { Spinner } from '.'
+
+import * as stories from './Spinner.stories'
+
+<Meta of={stories} />
+
+# Spinner
+
+Spinners provide a visual cue that an action is processing awaiting a course of change or a result.
+
+<StoryHeading label="Install" />
```

Comment:
StoryHeading has been deleted, you should rebase + replace with normal markdown headings :)

Reply:
Done!